### ISSUE TITLE
Type level programming defining kinds bug

### ISSUE THREAD

**Issue by razcore-rad** [JordanMartinez/purescript-jordans-reference#152]
## Context

1. Indicate the affected section (i.e. top-level folder)
Syntax

2. Indicate what kind of issue this is
Bug = a problem that needs to be corrected

3. If it applies, indicate the file to which you are referring using non-numerical-ordering names
Syntax//Type Level Programming Syntax/src/Defining Custom Kinds.purs

## Body

Since I'm looking at this, shouldn't this:
```purescript
-- To change the value-level type into a type-level type:
data Value_Level_Type
  = Value_Level_Instance1
  | Value_Level_Instance2
----------------------
-- We use this FFI-like syntax:
foreign import kind Value_Level_Type_Kind

-- and do not declare a right hand side (RHS) since it has no instances
--                    |--------- RHS -------|
--      data SomeType = Instance1 | Instance2
foreign import data Value_Level_Instance1_K :: Value_Level_Type_Kind
foreign import data Value_Level_Instance2_K :: Value_Level_Type_Kind
```

be rather this:

```purescript
-- To change the value-level type into a type-level type:
data Value_Level_Type
  = Value_Level_Instance1
  | Value_Level_Instance2
----------------------
-- We use this FFI-like syntax:
foreign import kind Value_Level_Type_Kind

-- and do not declare a right hand side (RHS) since it has no instances
--                    |--------- RHS -------|
--      data SomeType = Instance1 | Instance2
foreign import data Type_Level_Instance1 :: Value_Level_Type_Kind
foreign import data Type_Level_Instance2 :: Value_Level_Type_Kind
```
? Or something like that

**Comment by JordanMartinez** [comment]
mm... good point. 

I agree, and I think it might even be better to rename `Value_Level_Type_Kind` to just `Type_Level_Type` like this:
```purescript
-- To change the value-level type into a type-level type:
data Value_Level_Type
  = Value_Level_Instance1
  | Value_Level_Instance2
----------------------
-- We use this FFI-like syntax:
foreign import kind Type_Level_Type

-- and do not declare a right hand side (RHS) since it has no instances
--                    |--------- RHS -------|
--      data SomeType = Instance1 | Instance2
foreign import data Type_Level_Instance1 :: Type_Level_Type
foreign import data Type_Level_Instance2 :: Type_Level_Type
```

**Comment by JordanMartinez** [comment]
Closed by #151
